<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QmyVideoWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全屏</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="time_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="file_pushButton">
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_pushButton">
       <property name="text">
        <string>播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_pushButton">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="curr_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="curr_label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>音量：</string>
    </property>
   </widget>
   <zorder>pushButton_5</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>time_slider</zorder>
   <zorder>time_label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>curr_label</zorder>
   <zorder>curr_label_2</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
